src/Executables/Atlas/3. Configuration/5. Security/Core Isolation (VBS)/Current Configuration.ps1

```diff
@@ -18,26 +18,53 @@ if ($DisableAllVBS) {
 		New-ItemProperty -Path $memIntegrity -Name "Enabled" -Value 0 -PropertyType DWORD -Force
 		Remove-ItemProperty -Path $memIntegrity -Name "ChangedInBootCycle" -ErrorAction SilentlyContinue
 		Remove-ItemProperty -Path $memIntegrity -Name "WasEnabledBy" -ErrorAction SilentlyContinue
+		New-ItemProperty -Path $memIntegrity -Name "HVCIMATRequired" -Value 0 -PropertyType DWORD -Force
+		New-ItemProperty -Path $memIntegrity -Name "Locked" -Value 0 -PropertyType DWORD -Force
```

Comment:
What's the purpose of adding `HVCIMATRequired` and `Locked` and setting them to 0?
https://admx.help/?Category=Windows_10_2016&Policy=Microsoft.Windows.DeviceGuard::VirtualizationBasedSecurity

This also seems to be for the wrong key.